Formula/poetry.rb

```diff
@@ -247,6 +242,8 @@ def install
     # https://github.com/Homebrew/homebrew-core/pull/109777#issuecomment-1248353918
     ENV["PYTHON_KEYRING_BACKEND"] = "keyring.backends.null.Keyring"
 
+    ENV.prepend_path "PATH", Formula["python@3.11"].opt_libexec/"bin"
```

Comment:
Do users have to do this too to be able to use Poetry? If so we either need to add `caveats` or adjust the formula so they don't need to do this.

Reply:
yeah, this was indeed a bit surprising to me as well.